pyproject.toml

```diff
@@ -34,6 +34,7 @@ dependencies = [
     "numpy",
     "pydantic",
     "pymatgen>=2023.1.9",
+    "ase@git+https://gitlab.com/ase/ase.git@793179f4#egg=ase",
```

Comment:
Maybe also a part of the optional dependencies and to be added to strict?

Reply:
I think that would be better. I was not sure what level to put it on, @utf do you have a prefrence?

Reply:
Yes, put it in the optional dependencies under aims (you will need to create this) and also add it to struct with a specific version pinned.